WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -2421,28 +2368,30 @@ title_yabgu = {
 
 	allowed_to_grant = {
 		#OR = {
-		#	AND = {
+			AND = {
 				is_nomadic = yes
 				NOT = { culture = mongol }
-		#	}
-		#	AND = {
-		#		OR = {
-		#			is_feudal = yes
-		#			is_tribal = yes
-		#		}
-		#		NOT = {
-		#			OR = {
-		#				has_landed_title = e_mongol_empire
-		#				has_landed_title = e_golden_horde
-		#				has_landed_title = e_il-khanate
-		#			}
-		#		}
-		#		OR = {
-		#			religion = tengri_pagan
-		#			religion = tengri_pagan_reformed
-		#		}
-		#	}
-		#}
+			}
+			AND = {
+				OR = {
+					is_feudal = yes
+					is_tribal = yes
+					is_nomadic = yes
+				}
+				NOT = {
+					OR = {
+						has_landed_title = e_mongol_empire
+						has_landed_title = e_golden_horde
+						has_landed_title = e_il-khanate
+					}
+				}
+				OR = {
+					is_nomadic = yes
+					religion = tengri_pagan
+					religion = tengri_pagan_reformed
+				}
+			}
+		}
```

Comment:
@loup99 

I disagree. The conditions I wrote are simple:

1. Any steppe nomad can use steppe nomadic titles.
2. Sedentary Tengri Pagans can use steppe nomadic titles as a part of their religious heritage.

This ensures that the titles can continue to be used when a Tengri Steppe Nomad settles down. It allows for greater flavor as a Tengri Pagan.